localstack/services/events/event_bus.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+
+from localstack.aws.api.events import Arn, EventBusName, TagList
+from localstack.services.events.models_v2 import EventBus, RuleDict
+
+
+class EventBusWorker:
+    def __init__(
+        self,
+        name: EventBusName,
+        region: str,
+        account_id: str,
+        event_source_name: Optional[str] = None,
+        tags: Optional[TagList] = None,
+        policy: Optional[str] = None,
+        rules: Optional[RuleDict] = None,
+    ):
+        self.event_bus = EventBus(
+            name,
+            region,
+            account_id,
+            event_source_name,
+            tags,
+            policy,
+            rules,
+        )
+
+    @property
+    def arn(self):
+        return self.event_bus.arn
+
+
+EventBusWorkerDict = dict[Arn, EventBusWorker]
```

Comment:
I want to leave it as a separate type definition for now, because I want to use it in multiple times for the abstraction on top of the AccountRegionBundle to align e.g. EventBus the store data object and EventBusService the object so that the provider does not distinguish between both and only knows the EventBus as an object and the AccountRegionBundle handles the separation into data and pickling of it.

Reply:
ok, thanks for the explanation